### PR TITLE
Gift Entry: update elevate widget display

### PR DESCRIPTION
**Pull request by psadie-sfdo**
# Critical Changes

# Changes

# Issues Closed

# Community Ideas Delivered

# Features Intended for Future Release

# Features for Elevate Customers

# New Metadata

# Deleted Metadata

### CODE REVIEW

**Review by kenlewis-sfdo — APPROVED** [review]
@psadie-sfdo This looks great to me.  I don't have any initial comments - took a minute to grasp the transition idea in the new display state helper, but once I did it clicked.  Really nice work!  Please flip from draft to a regular PR.  Thanks Prince!

**Review by sokosaurus — APPROVED** [review]
Awesome work. Love to see all the refactoring you've done here!

File: src/lwc/geFormWidgetTokenizeCard/geFormWidgetTokenizeCard.html [hunk 1]
```diff
@@ -12,37 +12,41 @@
...
-                                    <template if:true={shouldDisplayEditPaymentInformation}>
-                                        <lightning-button variant="base"
-                                                          label={CUSTOM_LABELS.commonEditPaymentInformation}
-                                                          icon-name='utility:close'
-                                                          icon-position="right"
-                                                          onclick={disableReadOnlyMode}
-                                                          class='slds-button slds-button_base slds-float_right do-not-charge-card-button'
-                                                          data-qa-locator={qaLocatorEditPaymentInformation}>
-                                        </lightning-button>
+                                    <template if:true={isExpiredTransaction}>
```
Comment (sokosaurus) [inline src/lwc/geFormWidgetTokenizeCard/geFormWidgetTokenizeCard.html:32]: Hey @psadie-sfdo - should this conditional be scoped to include not only "Expired" transactions but also "Authorized" transactions as well? If the answers yes, lmk if I should make this change in my PR or yours. Thanks!

**Review by voduyemi — APPROVED** [review]
This looks good to me! Thanks @psadie-sfdo

**Pull request merged**